Sources/Maker.swift

```diff
@@ -565,6 +577,34 @@ public final class Maker {
         return self
     }
     
+    /// Creates edge relations for superview.
+    ///
+    /// - parameter insets: The insets for setting relations for superview.
+    ///
+    /// - parameter sides: The sides which will inculed from edge insets to setting relations.
+    ///
+    /// - returns: `Maker` instance for chaining relations.
+
+    @discardableResult public func edges(insets: UIEdgeInsets, sides: Set<Sides>) -> Maker {
+        sides.forEach { side in
```

Comment:
What's about removing old `edges` function and all `Sides.all` case and use it by default? Like:
```
@discardableResult 
public func edges(insets: UIEdgeInsets, sides: Set<Sides> = [.all]) -> Maker
```